ui/apps/platform/src/Containers/NetworkGraph/deployment/DeploymentSideBar.tsx

```diff
@@ -133,6 +132,7 @@ function DeploymentSideBar({
         );
     }
 
+    // id prop of heading corresponds to aria-labelledby prop of TopologySideBar
```

Comment:
Up to you if worthwhile to change. What do you think of an alternative to this comment:
- `labelledById` as required string prop on all sidebar components versus hard coded in component
- similar comment next to the prop type for this prop

It may help reduce the likelihood that these values ever get out of sync in a refactor, and maybe marginally increase the odds that a potential new sidebar component would be sure to include the same prop.